front/superzol-app/src/css/auth.css

```diff
@@ -0,0 +1,96 @@
+/* Root container settings */
+.root {
+    min-height: 100vh; /* Ensure the container takes up at least the full viewport height */
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    padding: 20px;
+    box-sizing: border-box;
+    background-color: #ecf0f1;
+}
+
+/* Paper container settings */
+.paper {
+    padding: 60px; /* Increased padding for more space inside */
+    border-radius: 20px;
+    background-color: rgba(255, 255, 255, 0.95);
+    backdrop-filter: blur(10px);
+    max-width: 500px;
+    width: 100%;
+    height: auto; /* Change height to auto to fit content */
+    box-sizing: border-box;
+}
+
+/* Body settings */
+body {
+    background-color: #ecf0f1;
+    margin: 0; /* Optional: Remove default margin for a full-page background */
+    font-family: Arial, sans-serif; /* Optional: Set a default font */
+}
+
+/* Form settings */
+.form {
+    width: 100%;
+    height: auto; /* Change height to auto to fit content */
+}
+
+/* Avatar settings */
```

Comment:
Please clean all the comments that are not really needed which is almost everything